app/bundles/EmailBundle/Form/Type/EmailSendType.php

```diff
@@ -186,6 +195,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 );
             }
         }
+        if (!empty($options['with_immediately'])) {
+            if ($this->coreParametersHelper->getParameter('mailer_spool_type') === 'file') {
+                $default = (isset($options['data']['immediately'])) ? $options['data']['immediately'] : false;
+                $builder->add(
+                'immediately',
```

Comment:
Move this and the lines that are adding params to the `add` method 4 spaces to the right.